Playlist_Genre_Jazz.php

```diff
@@ -1,12 +1,13 @@
 <?php
```

Comment:
all genre pages have the same changes as country - due to. copy and pasting the whole code but changing the queries for every single one